tasks/stats/index.js

```diff
@@ -0,0 +1,66 @@
+require('dotenv').load({silent: true})
+const console = require('better-console')
+const {GRAPHQL_ENDPOINT} = process.env
+const {GraphQLClient} = require('graphql-request')
+const fs = require('mz/fs')
+const path = require('path')
+const retry = require('async-retry')
+
+const client = new GraphQLClient(GRAPHQL_ENDPOINT)
+const STATS_FILEPATH = path.join(__dirname, path.normalize('../../src/metalsmith/settings/stats.json'))
+
+  // 'numberofmembers',
+  // 'amountpledged',
+  // 'amountpledgedwords',
+  // 'amountdonatedsofar',
+  // 'igivechart',
+  // 'igivechartwithoutother',
+  // 'citieschart',
+  // 'chaptermap',
+  // 'listofmembers'
```

Comment:
What's this?

Reply:
These are the names of the stats from the old Drupal site. I could probably kill them.

Reply:
Document it or kill it